examples/ShoppingAds/RejectMerchantCenterLink.php

```diff
@@ -0,0 +1,185 @@
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+            exit(1);
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID of the Google Ads account to reject the link request
+     * @param int $merchantCenterAccountId the Merchant Center account ID for the account requesting
+     *     to link
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $merchantCenterAccountId
+    ) {
+        $merchantCenterLinkService = $googleAdsClient->getMerchantCenterLinkServiceClient();
+
+        // Rejects a pending link request or unlinks an enabled link for a Google Ads account with
+        // $customerId from a Merchant Center account with $merchantCenterAccountId.
+        $response = $merchantCenterLinkService->listMerchantCenterLinks($customerId);
+        printf(
+            "%d Merchant Center link(s) found with the following details:%s",
+            $response->getMerchantCenterLinks()->count(),
+            PHP_EOL
+        );
+
+        foreach ($response->getMerchantCenterLinks() as $merchantCenterLink) {
+            /** @var MerchantCenterLink $merchantCenterLink */
+            printf(
+                "Link '%s' has status '%s'.%s",
+                $merchantCenterLink->getResourceName(),
+                MerchantCenterLinkStatus::name($merchantCenterLink->getStatus()),
+                PHP_EOL
+            );
+
+            // Checks if there is a link for the Merchant Center account we are looking for.
+            if ($merchantCenterAccountId === $merchantCenterLink->getIdUnwrapped()) {
+                // If the Merchant Center link is pending, reject it by removing the link.
+                // If the Merchant Center link is enabled, unlink Merchant Center from Google Ads by
+                // removing the link.
+                // In both cases, the remove action is the same.
+                self::removeMerchantCenterLink(
+                    $merchantCenterLinkService,
+                    $customerId,
+                    $merchantCenterLink
+                );
+                // There is only one MerchantCenterLink object for a given Google Ads account and
+                // Merchant Center account, so we can break early.
+                break;
+            }
+        }
+    }
+
+    /**
+     * Removes a Merchant Center link from a Google Ads client customer account.
+     *
+     * @param MerchantCenterLinkServiceClient $merchantCenterLinkServiceClient the
+     *     MerchantCenterLinkService client
+     * @param int $customerId the customer ID of the Google Ads account that has the link request
+     * @param MerchantCenterLink $merchantCenterLink the MerchantCenterLink object to remove
+     */
+    public static function removeMerchantCenterLink(
```

Comment:
This is intended to be a helper function, does this need to be public static in the class?